### PR TITLE
#156 - add Sixt provider

### LINKED ISSUE #156: Add Sixt provider
SIXT scooters should be available in Belgium, Germany, France and Spain: https://www.sixt.com/share/e-scooter/#/

image

### PR DESCRIPTION
**Pull request by AleksandraKozubal**
It should close #156.

### CODE REVIEW

File: app/Importers/SixtDataImporter.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        }
+
+        $existingCityProviders = [];
+        $countryName = "";
+
+        foreach ($this->sections as $section) {
+            $node = $section->parentNode->parentNode->parentNode;
+
+            foreach ($node->childNodes as $country) {
+                if ($country instanceof \DOMElement) {
```
Comment (kamilpiech97) [inline app/Importers/SixtDataImporter.php:55]: Maybe import that?

Comment (kamilpiech97) [reply app/Importers/SixtDataImporter.php:55]: And add `ext-dom` in `composer.json` file.

**Review by kamilpiech97 — CHANGES_REQUESTED** [review]

File: app/Importers/SixtDataImporter.php (hunk 1 above)
Comment (AleksandraKozubal) [reply app/Importers/SixtDataImporter.php:55]: If i would start with this node, then it would be harder to reach cities

Comment (AleksandraKozubal) [reply app/Importers/SixtDataImporter.php:55]: updated composer

**Review by EwelinaSkrzypacz — COMMENTED** [review]
`composer.json` file is not valid - please fix that

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Review by zmigrOO — APPROVED** [review]
Nice

**Pull request merged**